packages/block-editor/src/store/reducer.js

```diff
@@ -1072,7 +1180,8 @@ function selection( state = {}, action ) {
 			return { clientId: action.clientId };
 		case 'REPLACE_INNER_BLOCKS': // REPLACE_INNER_BLOCKS and INSERT_BLOCKS should follow the same logic.
 		case 'INSERT_BLOCKS': {
-			if ( ! action.updateSelection ) {
+			// REPLACE_INNER_BLOCKS can be called with an empty array.
+			if ( ! action.updateSelection || ! action.blocks.length ) {
```

Comment:
I'm pretty sure  this is  more impacting than we think here. Why not just pass "updateSelection" as false for this particulaar use-case?

Reply:
When the action is called with an empty array, it throws a fatal error, so I'm not sure how it would impact anything other than fixing a crash. Unless the crash was something else?

That said, I did actually _just_ start setting update selection to `false`, which also fixes some other things, so it probably isn't needed here any more.

Reply:
> When the action is called with an empty array, it throws a fatal error

Mmm, maybe that's the right fix then 🤷‍♂️

Reply:
Yeah, a few lines below you can see `return { clientId: action.blocks[ 0 ].clientId };`, so it tries to access the array even if there is nothing in it